skins/laika/src/scss/overrides.scss

```diff
@@ -1,63 +1,42 @@
 @import "variables";
 
-// Overriding Bulma variables
-
-// General page / body
-$primary: $fun-color-primary;
-$body-background-color: $fun-color-primary-lightest;
-
-// Text or font related
-$size-1: 1.7rem;
-$size-2: 1.6rem;
-$size-3: 1.5rem;
-$size-4: 1.4rem;
-$size-5: 1.3rem;
-$size-6: 1.2rem;
-$size-7: 1.1rem;
-
-$input-color: $fun-color-dark-lighter;
-$input-shadow: none;
-$input-focus-border-color: $fun-color-off-bright;
-$input-focus-box-shadow-size: 0;
-
-$text: $fun-color-dark;
-$family-primary: "Open Sans",Helvetica,Arial,sans-serif;
-$body-size: 14px;
-$body-line-height: 1.8;
-$danger: $fun-color-error;
-
-$title-size: $size-1;
-$title-weight: 400;
-$subtitle-size: $size-5;
-
-// Header / Navbar
-$navbar-height: 5.2rem;
-$navbar-box-shadow-size: 1px 2px 3px;
-$navbar-box-shadow-color: rgba(0, 0, 0, 0.1);
-$navbar-item-color: $text;
-$navbar-item-hover-color: $fun-color-primary;
-$navbar-item-hover-background-color: $fun-color-bright;
-$navbar-item-active-color: $fun-color-primary;
-$navbar-item-active-background-color: $fun-color-bright;
-
-$spacing-range-start: 0;
-$spacing-range-end: 54;
-$spacing-interval: 18;
-
 // Miscellaneous overrides
 
 .title {
 	margin-top: 30px;
 }
 
+.b-radio.radio {
+	height: 4em;
+	width: 320px;
+	border-bottom: 2px solid $fun-color-off-bright;
```

Comment:
Since the lines repeat in the navigation and maybe in the FAQ page, I'm wondering if it makes sense to introduce a "is-underlined" or "has-bottom-line" class. But I'm still unsure of our CSS language, so I don't know if it fits.

Reply:
I see what you mean, we could have some sort of class for 2px primary color underlines. I am wondering if it makes things unnecessary complicated, though I guess that's what we signed up for?

For example, the custom amount input field gets the same type of underline when it's valid but there are some additional rules attached to it, so if we had to attach an is-valid and is-underlined class, it would unnecessarily bloat up the class list. I guess a mixin would make more sense?!

We can also discuss this properly in person some time and come to an agreement where to draw the line with these classes, I am also not sure what the best practices are in this area.